portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -29,51 +29,45 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 import java.util.Arrays;
-import java.util.ArrayList;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+/** Servlet that returns comments*/
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-    ArrayList<String> words = new ArrayList<String>();
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    Query query = new Query("Task").addSort("timestamp", SortDirection.DESCENDING);
-    
+    //query for comment storage through datastore.
+    Query query = new Query("Task");
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
     PreparedQuery results = datastore.prepare(query);
     
-    //iterating over entities within Datastore
+    //populate comments ArrayList with entities within Datastore.
+    ArrayList<String> comments = new ArrayList<String>();
     for (Entity entity : results.asIterable()) {
-      long id = entity.getKey().getId();
       String title = (String) entity.getProperty("user-comments");
-      long timestamp = (long) entity.getProperty("timestamp");
+      comments.add(title);
     }
 
+    //covert comments to json for proper interpretation in script.js file.
```

Comment:
This comment is probably unnecessary, I think seeing the "response.setContentType("application/json")" already implies the response will want json.

Rationale: Someone reading these lines of code will probably understand fairly quickly the "why" when seeing the gson.toJson(comments) part.